Assets/Scripts/Enemy/IdleState.cs

```diff
@@ -20,7 +20,6 @@ void Start()
         wonderPosition = this.transform.position;
     }
 
-
     public override State StateFunction(UnityEngine.AI.NavMeshAgent agent)
```

Comment:
Can you remove all instances of `UnityEngine.AI.X`? Just import at the top `UnityEngine.AI`. It makes the code cleaner.